projects/ngrx.io/tools/examples/shared/package.json

```diff
@@ -41,11 +42,10 @@
     "zone.js": "~0.11.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "12.0.4",
-    "@angular/cli": "12.0.4",
-    "@angular/compiler-cli": "12.0.4",
-    "@angular/language-service": "12.0.4",
-    "@angular/platform-server": "12.0.4",
+    "@angular-devkit/build-angular": "13.0.1",
+    "@angular-eslint/eslint-plugin": "12.6.1",
+    "@angular-eslint/eslint-plugin-template": "12.6.1",
+    "@angular-eslint/template-parser": "12.6.1",
     "@ngrx/schematics": "^12.1.0",
```

Comment:
Sorry to do this, but these should also be updated 😅
As well as typescript (v4.4.4)

Reply:
No worries @timdeschryver. I haven't manually updated Angular in a while. `nx migrate` has been my friend!